app/darwinInit.js

```diff
@@ -99,10 +114,18 @@ if (process.platform === 'darwin') {
       } catch (e) {
       }
 
-      // relaunch and append argument expected in:
+      // if user has the Muon version of Brave as their default browser, let's
+      // ask the user to make Brave Core their new default. Passing this extra
+      // argument will prompt the user to ensure they agree with the choice
+      let defaultBrowserArg = ''
+      if (getDefaultBrowser() === 'com.electron.brave') {
```

Comment:
why can't we check default browser by https://github.com/brave/browser-laptop/blob/646212bafbce0b1bb8e93919435d27613b7af20c/app/index.js#L258?

Reply:
And by the method, you should be able to check default browser on Windows

Reply:
No check on Windows needed... but `defaultProtocols.every()` looks like a much nicer way to do it, thanks! 😄

Reply:
Fixed